packages/datacommons-mcp/datacommons_mcp/server.py

```diff
@@ -479,29 +431,40 @@ async def search_topics_and_variables(
             {
                 "topics": [
                     {
-                        "dcid": str,  # Topic DCID
+                        "dcid": str,  # Topic DCID (browse mode only)
                         "member_topics": list[str],  # Direct member topic DCIDs
                         "member_variables": list[str],  # Direct member variable DCIDs
-                        "places_with_data": list[str]  # Place DCIDs where data exists (only if place filtering was performed)
+                        "places_with_data": list[str]  # Place DCIDs where data exists (if place filtering was performed)
                     }
                 ],
                 "variables": [
                     {
                         "dcid": str,  # Variable DCID
-                        "places_with_data": list[str]  # Place DCIDs where data exists (only if place filtering was performed)
+                        "places_with_data": list[str]  # Place DCIDs where data exists (if place filtering was performed)
                     }
                 ],
-                "lookups": dict[str, str]  # DCID to name mappings
+                "lookups": dict[str, str],  # DCID to name mappings
+                "status": str  # Status of the search operation
             }
 
+        **Browse Mode**: Returns both topics and variables with hierarchical structure
+        **Lookup Mode**: Returns only variables (topics field is None)
+
     **Processing the Response:**
-    * **Topics**: Collections of variables and sub-topics. Use the lookups to get readable names.
+    * **Topics**: Collections of variables and sub-topics (browse mode only). Use the lookups to get readable names.
     * **Variables**: Individual data indicators. Use the lookups to get readable names.
     * **places_with_data**: Only present when place filtering was performed. Shows which requested places have data for each indicator.
     * **Filter and rank**: Treat all results as candidates and filter/rank based on user context.
     * **Data availability**: Use `places_with_data` to understand which places have data for each indicator.
+
+    **Best Practices:**
+    - Use **"browse"** when you want to understand data organization and discover related variables
```

Comment:
nit: maybe add the notion of "discover **collections** of variables or related variables"